test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/MapTest.scala

```diff
@@ -1011,11 +1110,13 @@ trait MapTest {
     assertEquals(TestObj(56), entry.getValue())
     assertEquals(TestObj(56), mp.get(key))
   }
-
 }
 
 object MapTest {
   final case class TestObj(num: Int)
+
+  def SIE[K, V](key: K, value: V): ju.Map.Entry[K, V] =
+    new ju.AbstractMap.SimpleImmutableEntry(key, value)
```

Comment:
Rebase accident? `SIE` is now defined once and for all in `Utils._`.

Reply:
Pretty sure unless it just merged clean. Rebase was pretty hard, missed some edits during and had to clean up after.